src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Analyzers.targets

```diff
@@ -62,6 +62,10 @@ Copyright (c) .NET Foundation. All rights reserved.
     <WarningLevel Condition="'$(Language)' == 'C#' And
                              '$(AnalysisLevel)' == 'preview'">9999</WarningLevel>
 
+    <!-- Intermediate step to enable ILLink.Analyzers so ILLink, Blazor, Xamarin, AOT, etc. can enable the same flags -->
+    <AnalyzeSingleFile Condition="'$(PublishSingleFile)' == 'true'">true</AnalyzeSingleFile>
+    <AnalyzeTrimming Condition="'$(PublishTrimmed)' == 'true'">true</AnalyzeTrimming>
```

Comment:
How should this interact with `SuppressTrimAnalysisWarnings`. As a user I would expect that `<AnalyzeTrimming>true</AnalyzeTrimming>` means I get analysis warnings.

Reply:
Since there is a section that wraps the analyzer trimming produced warnings, that would need to be changed to use AnalyzeTrimming and control AnalyzerWarnings. Otherwise, if AOT uses something different than PublishTrimmed the suppressions won't work. Thanks for the heads up! I will change the code :D